subgraphs/uniswap-forks/protocols/sushiswap/src/common/constants.ts

```diff
@@ -0,0 +1,8 @@
+////////////////////
+//////Versions//////
+////////////////////
+
+export const PROTOCOL_SUBGRAPH_VERSION = "1.1.1";
+export const PROTOCOL_METHODOLOGY_VERSION = "1.0.0";
+export const PROTOCOL_NAME = "Sushiswap";
```

Comment:
sushiswap is actually styled as `SushiSwap` with a capital `S`

Reply:
Updated